ci/prow/pj-on-kind.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
not sure if this is the copyright header we want, @adrcunha ?

Reply:
Indeed, this is the wrong boilerplate.

Reply:
Fixed. FWIW we catch boilerplate inconsistencies with a reusable presubmit check: https://github.com/kubernetes/repo-infra/blob/master/hack/verify_boilerplate.py
https://github.com/kubernetes/repo-infra#using-this-repository

Reply:
Interesting. We should make Matt's bot recognize bad boilerplates and suggest adding them (if it doesn't do it already).